.github/workflows/pkg-pr-new.yml

```diff
@@ -0,0 +1,31 @@
+name: Release for testing
+
+on:
+  pull_request:
+
+permissions:
+  contents: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  publish:
+    name: Publish to pkg.pr.new
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
Q: Why do we need all branches and tags here?

Reply:
Oops, I copied this in another CI and I didn't pay attention to this. :)